pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -797,7 +797,7 @@ public class Utils {
         Set<String> previousParameters = new HashSet<>()
 
         DeclarativeJobPropertyTrackerAction previousAction = null
-        WorkflowRun previousBuild = r.getPreviousBuild()
+        WorkflowRun previousBuild = r.getPreviousCompletedBuild()
```

Comment:
So that's a good question that I'd love your input on - I've got something similar at https://github.com/jenkinsci/workflow-multibranch-plugin/pull/65 as well. I honestly can't decide whether we should look at the last completed build or the last build regardless. In the `properties` step side, I have a check for whether `properties` had been run in the previous build (for reasons I explain there) so I thought completed would be better and made it the same here for consistent behavior, but I'm legitimately not sure which would be best.